doc/reference/testing.rst

```diff
@@ -111,6 +111,90 @@ supported.
+
+.. code-block:: python
+
+    from odoo.tests import TransactionCase, tagged
+
+    @tagged('-standard', 'nice')
+    class NiceTest(TransactionCase):
+        ...
+
+The tests defined in the above class will not be executed when using the ``--test-enable`` switch alone. To be run, the test must be selected explicitely:
+
+.. code-block:: text
+
+    $ odoo-bin --test-enable --test-tags nice
+
+With the above command, only the tests tagged ``nice`` are going to be executed.
+If you want to also execute the standard tests, you have to select them like this:
+
+.. code-block:: text
+
+    $ odoo-bin --test-enable --test-tags nice,standard
+
+The config switch parameter also accepts the ``+`` and ``-`` prefixes. The ``+`` prefix is implied and therefore, totaly optional. The ``-`` (minus) prefix is made to deselect tests tagged with the prefixed tags.
+
+For example, if we have a bunch of tests tagged as ``slow``, we can select the standard tests and disable the ``slow`` tests with this command:
+
+.. code-block:: text
+
+    $ odoo-bin --test-enable --test-tags 'standard,-slow'
+
+When you write a test that does not inherit from the Odoo BaseCase class, this test will not have the default tags, you have to add them explicitely to have the test included in the default test suite like this:
+
+.. code-block:: python
+
+    import unittest
+    from odoo.tests import tagged
+
+    @tagged('standard', 'at_install')
+    class SmallTest(unittest.TestCase):
+        ...
+
+Special tags:
+
+    - ``standard``: All Odoo tests that inherit from BaseCase are implicitely tagged standard. Also, the config switch ``--test-tags`` defaults to ``standard``. That means that an untagged test will be executed by default when tests are enabled.
+    - ``at_install``: Means that the test will be executed right after the module installation and before other modules are installed. This is a default implicit tag.
+    - ``post_install``: Means that the test will be executed after all the modules are installed. This is what you want for HttpCase tests most of the time.
+    - ``module_name``: All Odoo tests that inherit from BaseCase are implicitely marked with the technical name of the module as a tag. It means that you can easily include or exclude the module tests with a simple tag.
+
+Some examples:
+
+Run only the tests from the sale module:
+
+.. code-block:: text
+
+    $ odoo-bin --test-enable --test-tags sale
+
+Run the tests from the sale module but not the ones tagged as slow:
+
+.. code-block:: text
+
+    $ odoo-bin --test-enable --test-tags 'sale,-slow'
+
+Run only the tests from stock or tagged as slow (note the the ``-standard`` is not mandatory):
+
+.. code-block:: text
+
+    $ odoo-bin --test-enable --test-tags '-standard, slow, stock'
+
+Final remark about test tags, the ``@tagged`` decorator can only be used on classes, not on class methods.
```

Comment:
That would deserve being higher. And an other option would be to introduce `@tagged` using autodoc (and improving its docstring).